packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tabs/Tab.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components'
+import { Tab } from 'grommet'
+
+import { borderColor } from './colors'
+
+const StyledTab = styled(Tab)`
+  background-color: ${props => props.active ? 'white' : '#EFF2F5'}
```

Comment:
Same comment about the theming stuff from the current in use tabs: https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/components/TaskTabs/components/TaskTab/TaskTab.jsx#L13-L21